src/p5xr/core/p5xrInput.js

```diff
@@ -63,6 +64,18 @@ export default class p5xrInput {
     const p = this._pose?.transform?.position;
     return new p5.Vector(p.x, p.y, p.z);
   }
+  /** @returns {p5.Vector} Returns the current rotation as a Vector */
+  get rotation() {
+    this.updatePose();
+    if (this._pose){
+      const {x, y, z, w} = this._pose?.transform?.orientation;
+      let q = new Quaternion(x, y, z, w);
+      let e = q.toEuler();
+      let c = (p5.instance.angleMode == RADIANS)? 1 : 180 / Math.PI; // TODO: Change output with ANGLE MODE
```

Comment:
Better to use the private p5 helper [methods](https://github.com/processing/p5.js/blob/main/src/math/trigonometry.js#L345-L360) or [constants ](https://github.com/processing/p5.js/blob/main/src/core/constants.js#L184)for this.

Reply:
This function now checks for `angleMode `and uses p5's angle converter. I tested by using the manual test for input and changing the `angleMode `during set up.